oahray/bc-24-postit
# chore(tests): write tests for client side

### What does this PR do?
Includes client side tests, with some minor bug fixes.

### Description of Task to be completed?
Add tests for client side app.

### How should this be manually tested?
Run `npm run jest` on the command line. You should see the tests execute and pass.

### Any background context you want to provide?
Testing is an invaluable way to ensure quality.

### What are the relevant pivotal tracker stories?
Story #153260501
